src/addandremove.js

```diff
@@ -0,0 +1,105 @@
+const TaskContainer = document.querySelector('.list-Container');
+
+class CreatetodoList {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
+  }
+
+  static displayTasks = (task) => {
+    const listItem = document.createElement('li');
+    listItem.id = task.id;
+    listItem.className = 'lists';
+    listItem.innerHTML = `
+          <div class="list-Container">
+          <input type="checkbox" name="" id="${task.id}">
+          <p id="${task.id}">${task.description}</p>
+          </div>
+          
+          <div class="trash">
+          
+          <i id="pen" class="fa-solid fa-pen"></i>
+          <i id="delete" class="fa-solid fa-trash-can"></i>
+          </div>
+          `;
+    TaskContainer.appendChild(listItem);
+  };
+
+  static loadFromLocalStorage() {
+    let Tasks;
+
+    if (localStorage.getItem('TasksInfo')) {
+      Tasks = JSON.parse(localStorage.getItem('TasksInfo'));
+    } else {
+      Tasks = [];
+    }
+    return Tasks;
+  }
+
+  static displayTasksOnPage() {
+    const Tasks = CreatetodoList.loadFromLocalStorage();
+
+    Tasks.forEach((task) => {
+      CreatetodoList.displayTasks(task);
+    });
+  }
+
+  static removeBookFromPage(target) {
+    if (target.classList.contains('trash')) {
+      target.parentElement.remove();
+    }
+  }
+
+  static removeFromLocalStorage(element) {
+    let k = 0;
+    const Tasks = CreatetodoList.loadFromLocalStorage();
+
+    const idd = element.parentElement.id;
+    const newID = Number(idd);
+    for (let i = 0; i < Tasks.length; i += 1) {
+      if (Tasks[i].id === newID) {
+        k = i;
+        break;
+      }
+    }
+    Tasks.splice(k, 1);
+    let X = 1;
+    Tasks.forEach((task) => {
+      task.id = X;
+      X += 1;
+    });
+    localStorage.setItem('TasksInfo', JSON.stringify(Tasks));
+  }
+}
+
+export const removeItem = (e) => {
+  CreatetodoList.removeBookFromPage(e.target.parentElement);
+  CreatetodoList.removeFromLocalStorage(e.target.parentElement);
+};
+
+export const displayTasksOnWebPage = () => {
+  CreatetodoList.displayTasksOnPage();
+};
+
+export const addItem = () => {
+  const addInput = document.querySelector('.add-input');
+
+  if (addInput.value) {
+    const complete = false;
+
+    const loadTasks = CreatetodoList.loadFromLocalStorage();
+    const count = loadTasks.length + 1;
+    const newTask = new CreatetodoList(addInput.value, complete, count);
+
+    loadTasks.push(newTask);
+
+    CreatetodoList.displayTasks(newTask);
+    localStorage.setItem('TasksInfo', JSON.stringify(loadTasks));
+
+    // Reset input fields
+    addInput.value = '';
+  }
+};
```

Comment:
> By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).

-  You have done great so far. :+1: :+1: But there is one requirement you need to meet before moving to the next milestone. :+1: 
- [ ] As stated in the project, when a new task is created it needs to take the length of the todo task array as the index, currently, your index starts from zero (0), I'll suggest you update your logic so that your index starts from 1 as required in the project. :+1: :100:

Reply:
In the addItem function, I've updated the calculation of the count variable to start from 1 for new tasks. In the removeFromLocalStorage function, I've modified the logic to properly update the indices after removing a task from the array.